examples/darwin-framework-tool/commands/tests/TestCommandBridge.h

```diff
@@ -101,16 +106,50 @@ class TestCommandBridge : public CHIPCommandBridge,
         SetCommandExitStatus(err);
     }
 
-    /////////// DelayCommands Interface /////////
-    void OnWaitForMs() override
+    /////////// DelayCommands /////////
+    // This function is modifdied version in DelayCommands.cpp and is needed here in order to
+    // skip compilation of DelayCommands, because it requres SDK platform intialization.
+    CHIP_ERROR WaitForMs(
+        const char * _Nullable identity, const chip::app::Clusters::DelayCommands::Commands::WaitForMs::Type & value)
     {
-        dispatch_async(mCallbackQueue, ^{
+        dispatch_time_t delayTime = dispatch_time(DISPATCH_TIME_NOW, (int64_t)(value.ms * NSEC_PER_MSEC));
```

Comment:
```suggestion
        dispatch_time_t delayTime = dispatch_time(DISPATCH_TIME_NOW, ((int64_t)value.ms) * NSEC_PER_MSEC));
```
is probably better to avoid overflow....

Reply:
Updated

Reply:
I had to keep this as it was since dispatch_time is expecting int64_t and NSEC_PER_MSEC is  [UInt64](https://developer.apple.com/documentation/swift/uint64)

Reply:
> and NSEC_PER_MSEC is UInt64

That should be enough to avoid the overflow issue.